app/services/data_sources/adapters/google_sheets.py

```
@@ -0,0 +1,227 @@
+            f"https://sheets.googleapis.com/v4/spreadsheets/{spreadsheet_id}/values:batchGet",
+            params=[
+                ("majorDimension", "ROWS"),
+                *[("ranges", range_name) for range_name in ranges],
+            ],
+            timeout=10,
+        )
+        response.raise_for_status()
+
+        value_ranges = response.json().get("valueRanges", [])
+        datasets = []
+        for sheet, value_range in zip(sheets, value_ranges):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)